core/templates/pages/topic-editor-page/preview-tab/topic-preview-tab.component.html

```diff
@@ -0,0 +1,130 @@
+<div class="topic-preview-parent">
+  <div>
+    <background-banner></background-banner>
+  </div>
+  <div class="topic-preview-container">
+    <div class="topic-preview-nav">
+      <div class="topic-preview-navbar-tabs">
+        <div class="topic-preview-navbar-tab" ng-class="{'topic-preview-nav-active': $ctrl.activeTab === 'info'}" ng-click="$ctrl.changePreviewTab('info')">
+          <div class="topic-preview-icon">
+            <img alt="" class="tab-icon" ng-src="<[$ctrl.getStaticImageUrl('/icons/info_24px.svg')]>">
```

Comment:
let's add an alt attribute for this image. Ditto below in this file.

Reply:
It helps with accessibility eg. screen readers will announce the alt text for the image.

Reply:
Yes thanks. Will keep that in mind.